assets/site.sass

```diff
@@ -404,6 +410,42 @@ p.screenshot img
     text-shadow: 1px 1px 3px #000
     animation: 300ms ease-in-out fadeIn
     text-align: center
+    @media (prefers-reduced-motion: reduce)
+      animation: none
+
+  a
```

Comment:
Are you measing a { here?

Reply:
No, this is SASS, not SCSS

Reply:
I've used SASS on the website for years; was considering porting to SCSS, but it's basically the same except mandatory indentations and lack of {;} chars.

Reply:
This covers the differences: https://sass-lang.com/documentation/syntax